stable/external-dns/templates/deployment.yaml

```diff
@@ -302,6 +302,7 @@ spec:
         - name: azure-config-file
           {{- if or .Values.azure.secretName (and .Values.azure.resourceGroup .Values.azure.tenantId .Values.azure.subscriptionId) }}
           mountPath: /etc/kubernetes/
+          {{- else if .Values.azure.useManagedIdentityExtension }}
           {{- else }}
```

Comment:
What about the following approach?
```diff
-           {{- else if .Values.azure.useManagedIdentityExtension }}
-           {{- else }}
+ .         {{- else if not .Values.azure.useManagedIdentityExtension }}
```

Reply:
That reads a little clearer! I'll make a new commit.